src/CheatSheet.stories.mdx

```diff
@@ -0,0 +1,999 @@
+  />
+</div>
+
+<div class="layout">
+  <h4>
+    <a href="/docs/components-interactions--icon-button">Icon Button</a>
+  </h4>
+  <p>
+    Icon Button can be used in many different ways to indicate the ability to
+    add new content or items to a Collection.{' '}
+  </p>
+  <div>
+    <IconButton
+      btnStyle="primary"
+      data-testid="icon-button"
+      icon={{
+        children: [
+          {
+            attribs: {
+              d: 'M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z',
+            },
+            name: 'path',
+          },
+        ],
+        viewBox: '0 0 24 24',
+      }}
+      iconSize={20}
+      onClick={() => {}}
+      pixelButtonSize={40}
+    />
+  </div>
+</div>
+
+<div class="layout">
+  <h4>
+    <a href="/docs/components-interactions-button--button">Button</a>
+  </h4>
+  <p>
+    Button allow users to interact with the app and trigger actions. Buttons can
+    be used in a variety of ways in web apps, including Navigation, Form
+    submission, Call to Action, Confirmation or as Menu.
+  </p>
+  <Button data-testid="submit" id="testId" onClick={() => {}}>
+    Submit Button
+  </Button>
+</div>
+
+<div class="layout">
+  <h4><a href="/docs/components-interactions-date-picker--date-picker">DatePicker</a></h4>
+  <p>DatePicker allows users to select a date from a graphical calendar. It is commonly used in web apps to allow users to select a date for various purposes, such as scheduling appointments, setting deadlines, or filtering data.</p>
+  <DatePicker showDefaultInputIcon={true}
+  inputIconPosition= 'after'
+  datePickerStyle= 'primary'
+  datePickerSize='md'
+  invalidText= 'A valid value is required'
+  numberOfMonths={1}></DatePicker>
+
+<h4><a href="/docs/components-interactions-date-picker--date-picker-range">Date Picker Range</a></h4>
+
+  <DatePickerRange showDefaultInputIcon={true}
+  inputIconPosition= 'after'
+  datePickerStyle= 'primary'
+  datePickerSize='md'
+  invalidText= 'A valid value is required'
+  numberOfMonths={1}></DatePickerRange>
+</div>
+
+<div class="layout">
+<h4><a href="/docs/components-interactions-menu--menu-from-button">Menu</a></h4>
+<p>Menu refers to a Button or a Link that opens a drop-down menu with various options that the user can choose from.</p>
+<Menu items={[
+  {
+    label: 'Vanilla',
+    id: 'vanilla',
+    disabled: true,
+  },
+  { label: 'Chocolate', id: 'chocolate' },
+]}/>
+
+<h4><a href="/docs/components-interactions-menu--menu-from-link">Menu Link</a></h4>
+
+<MenuLink items={[
+  {
+    label: 'Vanilla',
+    id: 'vanilla',
+    disabled: true,
+  },
+  { label: 'Chocolate', id: 'chocolate' },
+]}/>
+</div>
+
+<div class="layout">
+<h4>
+<a href="/docs/components-interactions-checkbox--regular">Checkbox</a>
+</h4>
+<p>Checkbox allow users to select one or more options from a list. Checkboxes are typically used in situations where there are multiple options to choose from, and the user can select one or more options.</p>
```

Comment:
This is a bit wordy, I'd simply say "Checkboxes allow the user to make a binary, yes or no choice"